.github/workflows/release_portable_linux_packages.yml

```diff
@@ -181,6 +191,16 @@ jobs:
           --include "*.whl" \
           --include "*.tar.gz"
 
+      - name: Upload Releases to Staging S3
+        if: ${{ github.repository_owner == 'ROCm' }}
+        run: |
+          aws s3 cp ${{ env.DIST_ARCHIVE }} s3://${{ env.S3_BUCKET_TAR }}
```

Comment:
```suggestion
```
This is copied as part of the release step, here it gets written to the same bucket one more time.